src/io/flutter/editor/InlineWidgetViewModelData.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This file is also disabled behind a non-user configurable flag in the checked in code.